optuna/pruners/hyperband.py

```diff
@@ -150,15 +150,17 @@ def prune(self, study: "optuna.study.Study", trial: FrozenTrial) -> bool:
         bracket_study = self._create_bracket_study(study, i)
         return self._pruners[i].prune(bracket_study, trial)
 
-    def _calc_trial_allocation_budget(self, pruner_index: int) -> int:
+    def _calculate_trial_allocation_budget(self, pruner_index: int) -> int:
         """Computes the trial allocated budget for a bracket of ``pruner_index``.
 
         In the `original paper <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`, the
-        number of trials per one bracket is referred as ``n`` as calculated in Algorithm 1.
+        number of trials per one bracket is referred as ``n`` in Algorithm 1. Since we do not know
+        the total number of trials in the leaning scheme of Optuna, we calculate the ratio of the
+        number of trials here instead.
         """
 
         s = self._n_brackets - 1 - pruner_index
-        return self._n_brackets * (self._reduction_factor ** s) / (s + 1)
+        return self._n_brackets * (self._reduction_factor ** s) // (s + 1)
```

Comment:
This is now a floor division `//` (previously float division `/`). Although it should make little practical difference, how about using `math.ceil`, to mimic the paper?